cinema-soft/src/main/java/softserve/academy/cinemasoft/repository/CommentRepository.java

```diff
@@ -11,7 +11,5 @@
 public interface CommentRepository extends JpaRepository<Comment, Integer> {
 
 	List<Comment> findByMovie(Movie movie);
-
```

Comment:
You should still have separation between the methods.

Reply:
ok.